code/controllers/configuration/entries/game_options.dm

```diff
@@ -88,6 +88,12 @@
 	integer = FALSE
 	min_val = 0
 
+///how much does the amount of players get divided by to determine open assistants positions
+/datum/config_entry/number/assistant_scaling_coeff
+	config_entry_value = 8
```

Comment:
this would have to be 0 to have it be disabled unless enabled.

Reply:
ah yes i had it like this for testing, should i set it to zero?